src/app/assessment/assessment.component.html

```diff
@@ -2,14 +2,16 @@
   <ion-toolbar color="light">
     <ion-icon name="arrow-back" (click)="back()" color="primary" slot="start" margin-start></ion-icon>
     <ion-title text-center>Assessment</ion-title>
+    <ion-title *ngIf="saving || doAssessment || doReview" text-center class="subTitle">{{savingMessage}}</ion-title>
+    <ion-button *ngIf="doAssessment || doReview" [disabled]="savingButtonEnable" (click)="submit(true)" class="text-button" color="primary" slot="end" fill="clear" size="small">Save</ion-button>
```

Comment:
`[disabled]="savingButtonEnable"` it's either a wrong variable name or a wrong logic... which one do you choose :)

Reply:
And since the condition of `ngIf` for the text and button are the same, we can put the condition in a `<ng-container>` wrapper